app/views/api/v2/subnets/main.json.rabl

```diff
@@ -6,14 +6,8 @@ attributes :network, :network_type, :cidr, :mask, :priority, :vlanid, :gateway,
            :dns_primary, :dns_secondary,
            :from, :to, :created_at, :updated_at, :ipam, :boot_mode
 
-child :dhcp => :dhcp do
-  extends "api/v2/smart_proxies/base"
-end
-
-child :tftp => :tftp do
-  extends "api/v2/smart_proxies/base"
-end
-
-child :dns => :dns do
-  extends "api/v2/smart_proxies/base"
+Subnet.smart_proxies.keys.each do |proxy|
```

Comment:
It has to be `Subnet`, because `@object` would be `Subnet::Ipv4` for example. The smart proxies are registered only on `Subnet´.